src/shared/components/query/StudyListLogic.ts

```diff
@@ -7,347 +7,314 @@ import {computed, action} from "mobx";
-			if (filter && !meta.isCancerType)
-				for (let cancerTypes of [meta.ancestors, meta.priorityCategories])
-					for (let cancerType of cancerTypes)
-						map_node_filter.set(cancerType, true);
-		}
-		return map_node_filter;
-	}
-
-	@cached get map_node_filterBySelectedCancerTypes()
-	{
-		let map_node_filter = new Map<CancerTreeNode, boolean>();
-		if (this.store.selectedCancerTypes.length)
-		{
-			for (let cancerType of this.store.selectedCancerTypes)
-			{
-				let meta = this.getMetadata(cancerType);
-
-				// include selected cancerType and related nodes
-				map_node_filter.set(cancerType, true);
-				for (let nodes of [meta.ancestors, meta.descendantCancerTypes, meta.descendantStudies])
-					for (let node of nodes)
-						map_node_filter.set(node, true);
-			}
-		}
-		else
-		{
-			// include everything if no cancer types are selected
-			for (let node of this.store.treeData.map_node_meta.keys())
-				map_node_filter.set(node, true);
-		}
-		return map_node_filter;
-	}
-
-	@cached get map_node_filterBySelectedStudies()
-	{
-		let map_node_filter = new Map<CancerTreeNode, boolean>();
-		if (this.store.selectableSelectedStudies.length)
-		{
-			for (let study of this.store.selectableSelectedStudies)
-			{
-				let meta = this.store.treeData.map_node_meta.get(study) as NodeMetadata;
-
-				// include selected study and related nodes
-				map_node_filter.set(study, true);
-				for (let nodes of [meta.descendantStudies, meta.ancestors, meta.priorityCategories])
-					for (let node of nodes)
-						map_node_filter.set(node, true);
-			}
-		}
-		return map_node_filter;
-	}
-
-	getMetadata(node:CancerTreeNode)
-	{
-		return this.store.treeData.map_node_meta.get(node) as NodeMetadata;
-	}
-
-	@computed get mainView()
-	{
-		return new FilteredCancerTreeView(
-			this.store,
-			[
-				this.map_node_filterByDepth,
-				this.map_node_filterBySearchText,
-				this.map_node_filterBySelectedCancerTypes
-			]
-		);
-	}
-
-	@computed get cancerTypeListView()
-	{
-		return new FilteredCancerTreeView(
-			this.store,
-			[
-				this.map_node_filterByDepth,
-				this.map_node_filterBySearchText
-			]
-		);
-	}
-
-	@computed get selectedStudiesView()
-	{
-		return new FilteredCancerTreeView(
-			this.store,
-			[
-				this.map_node_filterByDepth,
-				this.map_node_filterBySelectedStudies,
-				{get: (node:CancerTreeNode) => !this.getMetadata(node).isPriorityCategory}
-			]
-		);
-	}
-
-	cancerTypeContainsSelectedStudies(cancerType:CancerType):boolean
-	{
-		let descendantStudies = this.getMetadata(cancerType).descendantStudies;
-		return _.intersection(this.store.selectableSelectedStudies, descendantStudies).length > 0;
-	}
-
-	getDepth(node:CancerType):number
-	{
-		let meta = this.getMetadata(node);
-		return meta.ancestors.length;
-	}
-
-	isHighlighted(node:CancerTreeNode):boolean
-	{
-		return !!this.store.searchText && !!this.map_node_filterBySearchText.get(node);
-	}
+export default class StudyListLogic {
```

Comment:
@adamabeshouse this file has very few actual changes.  i just finally changed it's line endings to unix style (from Andy's windows).   I marked where I changed